src/views/ColorPicker/ColorPicker.jsx

```diff
@@ -1,46 +1,24 @@
-import { useEffect, useState } from 'react';
 import Display from '../../components/Display/Display';
 import styles from './ColorPicker.css';
+import useAffirmations from '../../hooks/useAffirmation';
+import useColorPicker from '../../hooks/useColorPicker';
+
 
 export default function ColorPicker() {
-  const [fgColor, setFgColor] = useState('#ffcc00');
-  const [bgColor, setBgColor] = useState('#212121');
-  const [content, setContent] = useState('Hello, world!');
-  const [didChangeColor, setDidChangeColor] = useState(false);
-  const [affirmation, setAffirmation] = useState('');
+  //putting affirmation first so that the useColorPicker knows what bgcolor and fgcolor is from useAffirm
+  const affirmation = useAffirmations(bgColor, fgColor);
```

Comment:
this is correct! you just didn't have your `useAffirmations` hook set up to receive them